package.json

```diff
@@ -5,11 +5,11 @@
     "node": "8.9.4"
   },
   "dependencies": {
-    "babel-core": "6.26.0",
+    "babel-core": "^6.26.3",
```

Comment:
I think I'd prefer to pin these dependencies more explicitly, without the `^` specifier.

Reply:
sure, didn't notice that, good catch